cves/CVE-2010-1157.yml

```diff
@@ -126,8 +116,9 @@ discovered:
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
   answer: |
-    This issue was reported to the TomCat security team by a third party.
-  date: 2009-12-31
+    This vulnerability was discovered by Deniz Cevik, however, after searching through
+    bug reports and CVE statements, I was not able to find an exact date for this discovery.
+  date:
   automated:
   contest:
```

Comment:
Check me on this but werent automated and contest supposed to be filled either true or false?

Reply:
Updated